--- gosa-samba3.schema.old	2010-01-30 18:54:45.000000000 +0100
+++ gosa-samba3.schema	2010-01-30 16:29:10.000000000 +0100
@@ -395,6 +395,13 @@
        DESC 'Settings for gosa. Replaces parts of the gosa.conf. (v2.6)' SUP top STRUCTURAL
        MAY  ( gosaSetting ) )
 
+objectclass ( 1.3.6.1.4.1.10098.1.2.1.19.21
+       NAME 'gosaGroupOfURLs'
+       DESC 'Allow a group to be populated through a labeledURI values'
+       SUP top
+       AUXILIARY
+       MAY ( labeledURI ) )
+
 # GOto submenu entries
 objectclass (1.3.6.1.4.1.10098.1.2.1.43 NAME 'gotoSubmenuEntry'
         DESC 'GOto - contains environment settings (v2.6)' SUP top STRUCTURAL
